src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -1169,6 +1215,33 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
         private const val TOOL_WINDOW_SPLITTER_PROPORTION_KEY = "SNYK_TOOL_WINDOW_SPLITTER_PROPORTION"
         private const val NODE_INITIAL_STATE = -1
         private const val NODE_NOT_SUPPORTED_STATE = -2
+
+        private val CONTAINER_SCAN_COMMON_POSTFIX =
+            """
+                The plugin searches for Kubernetes workload files (*.yaml, *.yml) and extracts the used images.
+                During testing the image, the CLI will download the image
+                if it is not already available locally in your Docker daemon.<br><br>
+                If you are curious to know more about how the Snyk Container integration works, have a look at our
+                <a href="https://docs.snyk.io/features/integrations/ide-tools/jetbrains-plugins#analysis-results-snyk-container">docs</a>.
+            """.trimIndent()
+        val CONTAINER_SCAN_START_TEXT =
+            "Snyk Container scan for vulnerabilities.<br><br>${Companion.CONTAINER_SCAN_COMMON_POSTFIX}"
+        val CONTAINER_SCAN_RUNNING_TEXT =
+            "Snyk Container scan for vulnerabilities is now running.<br><br>${CONTAINER_SCAN_COMMON_POSTFIX}"
```

Comment:
Detekt :)

Reply:
imho false one, splitting that into two line will reduce readability

Reply:
Your choice, I think it would be better without the braces :). But up to you.

Reply:
You are right. I thought you're talking about string length. Will ix it